python/fledge/common/web/ssl_wrapper.py

```diff
@@ -80,21 +80,27 @@ def verify_against_revoked(cls):
     @classmethod
     def verify_against_ca(cls):
         echo_process = subprocess.Popen(['echo', cls.user_cert], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        a = subprocess.Popen(["openssl", "verify", "-CAfile", cls.ca_cert, "-x509_strict"], stdin=echo_process.stdout, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+        args = ["openssl", "verify", "-CAfile", cls.ca_cert]
+        cmd = "openssl verify -CAfile {}".format(cls.ca_cert)
+        # TODO: Add JIRA for to handle x509 strict check when open ssl version is 3 or so
+        # This is completely a workaround as of now CentOS9 stream has openssl version with 3.0
```

Comment:
; as of now CentOS Stream 9 has openssl version 3.0

Reply:
Will address in main branch